specs/AgriFood/Animal/doc/spec.md

```diff
@@ -84,19 +84,19 @@ The proposed model for the animal entity has the following properties:
             -   Type: DateTime
     -   Optional
 -   phenologicalCondition: Phenological condition of the animal
-    -   Attribute type: Text
+    -   Attribute type: [Text](https://schema.org/Text)
```

Comment:
we are missing here, `Property` (along the document)

Reply:
Fixed b64335a